docs/configuration-reference.md

```diff
@@ -20,8 +38,9 @@ An Accelerated Bridge CNI config with each field filled out looks like:
     "name": "some-net",
     "type": "accelerated-bridge",
     "deviceID": "0000:03:02.0",
+    "mac": "CA:FE:C0:FF:EE:00",
```

Comment:
a naive question.

where is the bridge name ?

Reply:
OK i assume there is logic which does the following VF -> PF -> Bridge 

wondering about VF lag

Reply:
Thanks, bridge name is lost in configuration ref. I will add it.

> OK i assume there is logic which does the following VF -> PF -> Bridge
No, bridge name should be explicitly set. `cni0` is default value.

Reply:
created an issue to sync config to README